components/org.wso2.carbon.privacy.forgetme.logs/src/main/java/org/wso2/carbon/privacy/forgetme/logs/instructions/LogFileInstruction.java

```diff
@@ -40,7 +40,7 @@
  */
 public class LogFileInstruction implements ForgetMeInstruction {
 
-    private static final Logger log = LoggerFactory.getLogger(LogFileInstruction.class);
+    private static final Log log = LogFactory.getLog(LogFileInstruction.class);
```

Comment:
Constants should be in all caps.

Reply:
Fixed with https://github.com/wso2/identity-anonymization-tool/pull/141